### ISSUE TITLE
train問題

### ISSUE THREAD

**Issue by aluds123** [wizyoung/YOLOv3_TensorFlow#158]
@wizyoung 您好:
        感謝你的分享，我想請教你2個問題
1.我在執行CUDA_VISIBLE_DEVICES=GPU_ID python train.py碰到如下的問題:
WARNING: Logging before flag parsing goes to stderr.
W0923 16:22:15.006506 140408668542720 lazy_loader.py:50]
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

W0923 16:22:15.007066 140408668542720 deprecation_wrapper.py:119] From train.py:24: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

Tensor("phase_train:0", dtype=bool)
W0923 16:22:15.018457 140408668542720 deprecation.py:323] From /usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/array_ops.py:1354: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
W0923 16:22:15.508944 140408668542720 deprecation.py:323] From train.py:41: py_func (from tensorflow.python.ops.script_ops) is deprecated and will be removed in a future version.
Instructions for updating:
tf.py_func is deprecated in TF V2. Instead, there are two
    options available in V2.
    - tf.py_function takes a python function which manipulates tf eager
    tensors instead of numpy arrays. It's easy to convert a tf eager tensor to
    an ndarray (just call tensor.numpy()) but having access to eager tensors
    means `tf.py_function`s can use accelerators such as GPUs as well as
    being differentiable using a gradient tape.
    - tf.numpy_function maintains the semantics of the deprecated tf.py_func
    (it is not differentiable, and manipulates numpy arrays). It drops the
    stateful argument making all functions stateful.

W0923 16:22:15.525203 140408668542720 deprecation_wrapper.py:119] From train.py:56: The name tf.data.Iterator is deprecated. Please use tf.compat.v1.data.Iterator instead.

W0923 16:22:15.525275 140408668542720 deprecation.py:323] From train.py:56: output_types (from tensorflow.python.data.ops.dataset_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.data.get_output_types(dataset)`.
W0923 16:22:15.525337 140408668542720 deprecation.py:323] From train.py:56: output_shapes (from tensorflow.python.data.ops.dataset_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.data.get_output_shapes(dataset)`.
W0923 16:22:15.526437 140408668542720 deprecation.py:323] From /usr/local/lib/python2.7/dist-packages/tensorflow/python/data/ops/iterator_ops.py:348: output_types (from tensorflow.python.data.ops.iterator_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.data.get_output_types(iterator)`.
W0923 16:22:15.526506 140408668542720 deprecation.py:323] From /usr/local/lib/python2.7/dist-packages/tensorflow/python/data/ops/iterator_ops.py:349: output_shapes (from tensorflow.python.data.ops.iterator_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.data.get_output_shapes(iterator)`.
W0923 16:22:15.526565 140408668542720 deprecation.py:323] From /usr/local/lib/python2.7/dist-packages/tensorflow/python/data/ops/iterator_ops.py:351: output_classes (from tensorflow.python.data.ops.iterator_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.data.get_output_classes(iterator)`.
W0923 16:22:15.527760 140408668542720 deprecation_wrapper.py:119] From train.py:74: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

W0923 16:22:15.672477 140408668542720 deprecation_wrapper.py:119] From /home/aluds/YOLOv3_TensorFlow-master_test/utils/layer_utils.py:27: The name tf.nn.max_pool is deprecated. Please use tf.nn.max_pool2d instead.

W0923 16:22:16.259958 140408668542720 deprecation_wrapper.py:119] From /home/aluds/YOLOv3_TensorFlow-master_test/utils/layer_utils.py:116: The name tf.image.resize_nearest_neighbor is deprecated. Please use tf.compat.v1.image.resize_nearest_neighbor instead.

W0923 16:22:16.650896 140408668542720 deprecation_wrapper.py:119] From /home/aluds/YOLOv3_TensorFlow-master_test/model.py:265: The name tf.log is deprecated. Please use tf.math.log instead.

W0923 16:22:17.176789 140408668542720 deprecation_wrapper.py:119] From train.py:80: The name tf.losses.get_regularization_loss is deprecated. Please use tf.compat.v1.losses.get_regularization_loss instead.

W0923 16:22:19.372805 140408668542720 deprecation_wrapper.py:119] From train.py:85: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.

W0923 16:22:19.587430 140408668542720 deprecation_wrapper.py:119] From train.py:88: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

W0923 16:22:19.591284 140408668542720 deprecation_wrapper.py:119] From train.py:96: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

W0923 16:22:19.596576 140408668542720 deprecation_wrapper.py:119] From /home/aluds/YOLOv3_TensorFlow-master_test/utils/misc_utils.py:145: The name tf.train.piecewise_constant is deprecated. Please use tf.compat.v1.train.piecewise_constant instead.

W0923 16:22:19.615684 140408668542720 deprecation_wrapper.py:119] From /home/aluds/YOLOv3_TensorFlow-master_test/utils/misc_utils.py:153: The name tf.train.MomentumOptimizer is deprecated. Please use tf.compat.v1.train.MomentumOptimizer instead.

W0923 16:22:19.615755 140408668542720 deprecation_wrapper.py:119] From train.py:112: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

Saving optimizer parameters to checkpoint! Remember to restore the global_step in the fine-tuning afterwards.
2019-09-23 16:22:22.941943: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcuda.so.1
他有開始訓練，不過不知道這個warnning會不會成問題

2.我打算要修改layer_utils，想請問在args.py的restore_path那邊要如何修改(該如何存成新的ckpt檔案?)

麻煩大家指教，謝謝
